lib/bindings/MQTTBinding.js

```diff
@@ -272,15 +352,19 @@ function start(newConfig, callback) {
 }
 
 function stop(callback) {
-    logger.info('Stopping IoT Agent');
+    logger.info('Stopping MQTT Binding' + (new Date()));
```

Comment:
Is it really needed to log the current date and time since already included in the `time` property of the message logged by `logops`?